sass/components/_modal.scss

```diff
@@ -240,8 +240,9 @@
     }
 
     .modal-header {
+        display: flex;
         min-height: 56px;
-        padding: 14px 15px 11px;
+        padding: 14px 48px 11px 15px;
```

Comment:
whats that massive padding-right doing? And why is it begin reset on mobile?

Is this for spacing out the button? If yes, this could be done better since this will break on translations with longer texts in the buttons.

Reply:
It's so that the button/title doesn't overlap the close button in desktop view since the close button still has `float: right` applied to it. Previously, that padding was actually a margin on the button (you can see the old margin above [this comment](https://github.com/mattermost/mattermost-webapp/pull/9975#discussion_r822379007) but it only applied if there was a non-close button in the modal header.

The reason it's not needed in mobile view is because the rest of the header styling is different there. If you check out the screenshots above, the text and close button are on a separate line from the add button. When I had that padding in mobile view, the button wouldn't stretch across the full width available.

Reply:
just as i thought ... still pretty hacky! :D

Reply:
Oh yeah, 100%. Unfortunately, we're still bound by our ancient version of Bootstrap